src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiHplManifestSpec.groovy

```diff
@@ -13,12 +13,20 @@ class JpiHplManifestSpec extends Specification {
         setup:
         project.with {
             apply plugin: 'jpi'
+            dependencies.with {
+                implementation('org.apache.commons:commons-lang3:3.9')
```

Comment:
I would have hoped for an integration test.

Reply:
I agree, but didn't want to hold this up for that change. I've created https://github.com/jenkinsci/gradle-jpi-plugin/issues/136 to track this. Thanks!